pkg/kubelet/network/cni/cni.go

```diff
@@ -266,6 +266,9 @@ func (plugin *cniNetworkPlugin) GetPodNetworkStatus(namespace string, name strin
 	if err != nil {
 		return nil, fmt.Errorf("CNI failed to retrieve network namespace path: %v", err)
 	}
+	if netnsPath == "" {
+		return nil, fmt.Errorf("Cannot find network namespace for the terminated container %q", id)
```

Comment:
Yes, if you had `--v=4` you would see this message twice (once as info and ultimately once as a warning).

Maybe changing this to something like `Cannot find the network namespace, skipping pod network status` ?